runtime/doc/api.txt

```diff
@@ -1949,12 +1949,35 @@ nvim_get_option_info({name})                          *nvim_get_option_info()*
     • commalist: List of comma separated values
     • flaglist: List of single char flags
 
+    The last set information applies to the local value in the current buffer
+    or window if it is available, otherwise the global value information is
+    returned.
+
     Parameters: ~
       • {name}  Option name
 
     Return: ~
         Option Information
 
+nvim_get_option_info_ex({name}, {*opts})           *nvim_get_option_info_ex()*
```

Comment:
This is probably not the best name but I was unable to come up with anything better. Any suggestions are welcome.

Or maybe we could make the opts table optional and have only one function nvim_get_option_info, at least on the Lua side?

Reply:
I suggest `nvim_get_option2`. See also https://github.com/neovim/neovim/issues/20451

Reply:
Wouldn't `nvim_get_option_info2` make more sense?

Reply:
I will change that to `nvim_get_option_info2`. `nvim_get_option2` suggests that option value is returned, while it is not.

Reply:
Done